tests/test_tokenization_fast.py

```diff
@@ -1,23 +1,52 @@
 import logging
```

Comment:
Needs compyright here.

The diff for this file is slightly complicated to read, did you wrap the tests in subtests, iterating over every tokenizer? Is that better than doing a mixin like we do in other test classes?

Reply:
I kinda kept the original setup made by @mfuntowicz  even though I agree switching to a mixin would probably be easier to read in the end.